File: frontend/src/dto/edit/EditTaskSeriesDTO.ts
import type {Priority} from "@/dto/Priority.ts";
import type {Status} from "@/dto/Status.ts";

export type EditTaskSeriesDTO = {
    name: string;
    itemId: string[];
    assignedTo: string[];
    priority: Priority;
    status: Status;
    dueDate: string;
    repetition: number;
    homeId: string;
}

File: backend/src/main/java/org/example/backend/controller/dto/edit/EditTaskSeriesDTO.java
package org.example.backend.controller.dto.edit;

import org.example.backend.domain.task.Priority;
import org.example.backend.domain.task.Status;

import java.time.LocalDate;
import java.util.List;

public record EditTaskSeriesDTO(String name,
                                List<String> itemId,
                                List<String> assignedUser,
                                Priority priority,
                                Status status,
                                LocalDate dueDate,
                                int repetition,
                                String homeId) {
}
